internal/controllers/common/resource_definitions/resource_definitions.go

```diff
@@ -673,6 +673,28 @@ func NewPodForCR(cr *model.CryostatInstance, specs *ServiceSpecs, imageTags *Ima
 		volumes = append(volumes, eventTemplateVolume)
 	}
 
+	// Add any ProbeTemplates as volumes
+	for _, template := range cr.Spec.ProbeTemplates {
+		probeTemplateVolume := corev1.Volume{
+			Name: "template-" + template.ConfigMapName,
```

Comment:
We should use a different prefix than the event templates to avoid the possible of identically named volumes, `probe-template-` would work.